.github/workflows/test.yml

```
@@ -18,7 +18,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.8", "3.9", "3.10", "3.11"]
+        python-version: ["3.11"]
```

```suggestion
        python-version: ["3.11", "3.12", "3.13"]
```

if we're trimming versions at one end (which is fine), we should add them at the other